tomcat-base/src/shared/main/java/eu/xenit/alfresco/tomcat/embedded/share/tomcat/ShareTomcatFactoryHelper.java

```diff
@@ -22,45 +18,21 @@ public static void createShareConfigCustomFile(ShareConfiguration shareConfigura
         try {
             Path inputPath = Paths.get(shareConfiguration.getShareConfigTemplateFile());
```

Comment:
There is no point in copying a template file to another location here. The share-config-custom.xml should be put directly in the right location.